### PR TITLE
Fix term and conditions handled as a malformed permalink

### PR DESCRIPTION
**Pull request by Florian14**
#321

### CODE REVIEW

**Comment by ClaireGizard** [comment]
LGFM

**Review by ClaireGizard — APPROVED** [review]

File: vector/src/main/java/im/vector/app/features/home/HomeActivity.kt [hunk 1]
```diff
@@ -220,7 +220,9 @@ class HomeActivity :
                         }
                         is HomeActivitySharedAction.InviteByEmail      -> Unit // no-op
                         HomeActivitySharedAction.OpenTermAndConditions -> {
-                            openUrlInChromeCustomTab(this, null, VectorSettingsUrls.TAC)
+                            // Tchap: Term and conditions are detected as permalinks (same structure), so open it in a WebView
```
Comment (giomfo) [inline vector/src/main/java/im/vector/app/features/home/HomeActivity.kt:223]:
```suggestion
                            // Tchap: the Term And Conditions url is detected as a permalink (same prefix), which make the application fail to open it from ChromeCustomTab, so we open it here directly in a WebView
```

File: vector/src/main/java/im/vector/app/features/settings/VectorSettingsHelpAboutFragment.kt [hunk 2]
```diff
@@ -87,7 +88,9 @@ class VectorSettingsHelpAboutFragment @Inject constructor(
         // terms & conditions
         findPreference<VectorPreference>(VectorPreferences.SETTINGS_APP_TERM_CONDITIONS_PREFERENCE_KEY)!!
                 .onPreferenceClickListener = Preference.OnPreferenceClickListener {
-            openUrlInChromeCustomTab(requireContext(), null, VectorSettingsUrls.TAC)
+            // Tchap: Term and conditions are detected as permalinks (same structure), so open it in a WebView
```
Comment (giomfo) [inline vector/src/main/java/im/vector/app/features/settings/VectorSettingsHelpAboutFragment.kt:91]:
```suggestion
            // Tchap: the Term And Conditions url is detected as a permalink (same prefix), which make the application fail to open it from ChromeCustomTab, so we open it here directly in a WebView
```

**Review by giomfo — APPROVED** [review]
commented and approved
SGTM

**Pull request merged**